src/wp-includes/rest-api/endpoints/class-wp-rest-font-families-controller.php

```diff
@@ -0,0 +1,554 @@
+<?php
+/**
+ * REST API: WP_REST_Font_Families_Controller class
+ *
+ * @package WordPress
+ * @subpackage REST_API
+ * @since 6.5.0
+ */
+
+/**
+ * Font Families Controller class.
+ *
+ * @since 6.5.0
+ */
+class WP_REST_Font_Families_Controller extends WP_REST_Posts_Controller {
+	/**
+	 * Whether the controller supports batching.
+	 *
+	 * @since 6.5.0
+	 * @var false
+	 */
+	protected $allow_batch = false;
+
+	/**
+	 * Checks if a given request has access to font families.
+	 *
+	 * @since 6.5.0
+	 *
+	 * @param WP_REST_Request $request Full details about the request.
+	 * @return true|WP_Error True if the request has read access, WP_Error object otherwise.
+	 */
+	public function get_items_permissions_check( $request ) { 
```

Comment:
```suggestion
	public function get_items_permissions_check( $request ) {
```

Whitespace at end of line is not allowed

Reply:
Updated in https://github.com/WordPress/wordpress-develop/pull/6034/commits/a0836f0f6b4fffc3259a2711f2ea43b02823a6e2